src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -104,34 +103,38 @@
         options.enableCrossPartitionQuery(true);
         options.maxDegreeOfParallelism(2);
 
-        int skipCount = 0;
-        int takeCount = 10;
-
         ObjectMapper objMapper = ObjectMapperFactory.getObjectMapper();
 
+        String _q = "";
+        
+        if(q.isPresent()) {
+            if(q.get() != null && q.get() != "") {
+                _q = " and contains(m.textSearch, '" + q.get() + "') ";
+            }
+        }
+
+        String queryString = "select m.id, m.partitionKey, m.actorId, m.type, m.name, m.birthYear, m.deathYear, m.profession, m.textSearch, m.movies from m where m.type = 'Actor'  " + _q + " OFFSET "  + _pageNumber + " LIMIT " + _pageSize;
```

Comment:
Let's move query templates into private static finals of classes.  Additionally, we can put placeholders in the string and use String.format in the data access layer.

Reply:
Same as above. Will perform all of the refactoring as part of #93. Items #95 and #96 cover all the concerns raised.

Reply:
This is actually fixed in the latest commit. The change says outdated. Please see the latest commit.